### PR TITLE
feat(UI): Save dashboard changes

### PR DESCRIPTION
**Pull request by Thebarda**
## Description

This saves the dashboard

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [x] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [x] 23.10.x (master)

<h2> How this pull request can be tested ? </h2>

- Go to the Dashboards page
- Click on a created dashboard
- Change the dashboard widget(s)
- Click on the save button
- -> The dashboard is saved and you are back in the view mode

#### Community contributors & Centreon team

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -17666,3 +17666,9 @@ msgstr "Si vous quittez le mode édition, vos changements ne seront pas sauvegar
 
 msgid "If you leave {{dashboardName}}, your changes will not be saved."
 msgstr "Si vous quittez {{dashboardName}}, vos changements ne seront pas sauvegardés."
+
+msgid "Your dashboard has been saved"
+msgstr "Votre tableau de bord a été enregistré"
```
Comment (smau-centreon) [inline centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:17671]:
```suggestion
msgstr "Votre tableau de bord a été sauvegardé"
```

**Review by smau-centreon — CHANGES_REQUESTED** [review]

**Review by smau-centreon — APPROVED** [review]

File: centreon/www/front_src/src/Dashboards/api/endpoints.ts [hunk 2]
```diff
@@ -1,3 +1,3 @@
-export const baseEndpoint = './api/latest';
+export const baseEndpoint = 'http://localhost:5005/centreon/api/latest';
```
Comment (g10) [inline centreon/www/front_src/src/Dashboards/api/endpoints.ts:1]:
```suggestion
export const baseEndpoint = './api/latest';

```

File: centreon/packages/ui/src/components/Modal/Modal.stories.tsx [hunk 3]
```diff
@@ -62,7 +65,8 @@ export const AsDangerAction: Story = {
         isDanger
         labels={{
           cancel: 'Cancel',
-          confirm: 'Confirm'
+          confirm: 'Confirm',
+          loading: 'Loading'
```
Comment (g10) [inline centreon/packages/ui/src/components/Modal/Modal.stories.tsx:69]:
```suggestion
          confirm: 'Confirm'
```

File: centreon/packages/ui/src/components/Modal/Modal.stories.tsx [hunk 4]
```diff
@@ -13,7 +13,10 @@ const meta: Meta<typeof Modal> = {
       options: ['small', 'medium', 'large']
     }
   },
-  component: Modal
+  component: Modal,
+  parameters: {
+    chromatic: { delay: 300 }
+  }
```
Comment (g10) [inline centreon/packages/ui/src/components/Modal/Modal.stories.tsx:19]:
```suggestion
  component: Modal
```

i assume the chromatic delay was for the loading state? which has been removed

Comment (Thebarda) [reply centreon/packages/ui/src/components/Modal/Modal.stories.tsx:19]: Yeah right

**Pull request merged**